.github/workflows/conda-cpp-tests.yaml

```diff
@@ -97,7 +97,7 @@ jobs:
   tests:
     needs: compute-matrix
     strategy:
-      fail-fast: false
+      fail-fast: true
```

Comment:
Let's add this option explicitly to the wheel jobs, where it is currently using the default value of `true`. Just want to be explicit everywhere.

Reply:
It's already there.